README.md

```diff
@@ -1,137 +1,137 @@
 # What Is Crate?
 
-Crate is a fast, scalable, easy to use SQL database that plays nicely
-with containers like Docker. It feels like the SQL databases you know,
-however makes scaling and operating your database ridiculously easy --
-regardless of the volume, complexity, or type of data. Crate is open
-source. It ingests millions of records per second for time series
-setups and delivers analytics results in subsecond real time.
-
-It comes with a distributed sort and aggregation engine, fast multi
-index queries, native full-text search and super simple scalability
-with sharding and partitioning builtin already. Preconfigured
-replication takes care of data resiliency. The cluster management can
-easily supervised with its builtin admin UI. Crate's masterless
-architecture and simplicity make the data part of Docker environments
+Crate is an open source fast, scalable, easy to use SQL database that plays
+nicely with containers like Docker. It feels like the SQL databases you know,
+but makes scaling and operating your database ridiculously easy - regardless of
+the volume, complexity, or type of data. It ingests millions of records per
+second for time series setups and delivers analytics results in sub-second real
+time.
+
+Crate comes with a distributed sort and aggregation engine, fast multi index
+queries, native full-text search and super simple scalability with sharding and
+partitioning builtin. Preconfigured replication takes care of data resiliency.
+The cluster management can be supervised with a built-in admin UI. Crate's
+masterless architecture and simplicity make the data part of Docker environments
 easy and elegant.
 
-Crate provides several installation packages, including a supported
-Docker image. It fits perfectly in an orchestrated microservices
-environment. It acts like an ether, an omnipresent, persistent layer
-for data. This way, application containers access their data
-regardless on which host the data nodes run.
+Crate provides several installation packages, including a supported Docker
+image. It fits perfectly into an orchestrated microservices environment. It acts
+like an ephemeral, omnipresent, persistent layer for data. Application
```

Comment:
Erm, an *ephemeral*, persistant layer is a contradiction ... :) Maybe the "ether" metapher is confusing in this context?! I understand it as "all-pervasive", "everywhere".